bg_simulator.py

```diff
@@ -58,7 +55,7 @@ def assign_carbs(sugar, last_carbs, sugar_in_range):
         carbs = random.triangular(-50, 100, 60)
     return carbs
 
-def stitch_func(num_days=180):
+def simulate(num_days=180):
```

Comment:
It's confusing that in the current state of the project, you're defaulting to 10 days for fast development purposes but the default here is 180. If you're going to change the default for development purposes, please do it here instead of wherever else it's being overridden currently.